distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -854,7 +886,7 @@ type Table
                  table = Examples.inventory_table
                  double_inventory = table.at "total_stock" * 2
                  table.set "total_stock" double_inventory
-    set : Text -> Column.Column | Vector.Vector -> Table
+    set : Text -> Column | Vector.Vector -> Table
```

Comment:
This change is wrong: `Column` is the module, `Column.Column` is the type.

But as am cleaning up a lot of this in next PR not worth blocking this merge over.

Reply:
Okay, I think I was also trying to change the imports but then reverted that and this may be a leftover.